src/Component/NavBar/SignUp.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { useState } from "react";
+import Alert from "@mui/material/Alert";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import Input from "@mui/material/Input";
+import InputLabel from "@mui/material/InputLabel";
+import InputAdornment from "@mui/material/InputAdornment";
+import FormControl from "@mui/material/FormControl";
+import Visibility from "@mui/icons-material/Visibility";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import LoginIcon from "@mui/icons-material/Login";
+import { Container } from "@mui/material";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Chip from "@mui/material/Chip";
+import FaceIcon from "@mui/icons-material/Face";
+import Link from "@mui/material/Link";
+import { Route, Routes } from "react-router-dom";
+import { Link as RouterLink } from "react-router-dom";
+import Grid from "@mui/material/Grid2";
+
+const isEmail = (email) =>
+  /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(email);
+
+function SignUp() {
+  const [showPassword, setShowPassword] = useState(false);
+  const [usernameInput, setUsernameInput] = useState();
```

Comment:
I don't believe the user model/schema in the backend has a username field. So we should either take it out or talk with backend team about adding it on their end